bring/src/main/java/com/bobocode/petros/bring/exception/ExceptionMessage.java

```diff
@@ -0,0 +1,14 @@
+package com.bobocode.petros.bring.exception;
+
+/**
+ * Provides error messages in a single place.
+ */
+public enum ExceptionMessage {
```

Comment:
could be an interface
formatting

Reply:
> could be an interface formatting

I started to write an agree message regarding that comment, but then I thought that Vlad did that for some reason and it looks like it's kind of best practice ;) 
https://stackoverflow.com/questions/25485057/use-enum-for-keeping-constants
And that's something new for me. Cool !